app/models/imageable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: imageables
+#
+#  id             :bigint           not null, primary key
+#  imageable_type :string           not null
+#  picture        :string
+#  type           :integer
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  imageable_id   :bigint           not null
+#
+# Indexes
+#
+#  index_imageables_on_imageable  (imageable_type,imageable_id)
+#
+class Imageable < ApplicationRecord
```

Comment:
I think the class should be just Image? Then the polymorphic association is the imageable one

Reply:
Much better!

Reply:
Forgot about this